TESTING.md

```diff
@@ -476,20 +477,48 @@ To open the context menu:
 2. Create new chat
 3. User doesn't see model dropdown
 
-### Enterprise account
+### Cody Enterprise - Cody Gateway Instance
 
 #### Chat
 
-1. Login to Cody Enterprise account
-2. Create new chat
-3. User should see the default model in the dropdown but is unable to change it
+1. Login to the Cody Enterprise Gateway Instance
+2. Create a new chat
+3. User should not be able to change the LLM model on the chat window
+4. Ask Cody a question in the new chat window. 
 
 #### Commands
 
-1. Login to Cody Enterprise account
+1. Login to the Cody Enterprise Cody Gateway Instance
 2. Go to commands panel
 3. Trigger command
-4. Command should be executed with default model
+4. Command should be executed
+
+#### Completions
+
+1. Login to the Cody Enterprise Cody Gateway Instance
+2. Generate a completion using the manual-trigger key binding on any primary languages (Javascript, Typescript, TypescriptReact, Python, Go).
```

Comment:
```suggestion
2. Generate a completion using the manual-trigger key binding on any primary languages (Javascript, Typescript, TypescriptReact, Python, Go). A completion should appear.
```

@aramaraju

- Document the keyboard shortcut to press or the name of the command so the testers can look up the keyboard shortcut.
- This test should specify having a particular repo cloned and open, and mention specific files to open. It will make testing easier and make reproducing any bugs much easier.

@pkukielka does the tester's setup include other autocomplete providers? If so we need to ensure the completions are coming from Cody specifically somehow.

Reply:
`opt \` is the manual trigger hotkey for autocomplete. 

The QA testers have experience with this, I don't think we need to assume they have another completion provider. The Cody logo also "spins/loads" when it triggers a completion, so theres indication from the UI that it's coming from Cody